### ISSUE TITLE
populate erc20 token table

### ISSUE THREAD

**Issue by jrhite** [omisego/elixir-omg#1105]
blocked on: https://github.com/omisego/elixir-omg/issues/1033

Initial thoughts: when a deposit comes in, start a new async process that will look up the contract address of the deposit and if it's not in our ERC20 table, then fetch the ERC20 symbol and data from somewhere. The deposit itself does not block and continues on as usual.

**Comment by unnawut** [comment]
Given now we're handling bugs (although indirectly) from code and DB fields that are not being used, I'll close this DB-only issue so we don't need to maintain what we don't use.

Feel free to submit a new issue when a user story arises and is promoted into an engineering cycle!
